custom-fonts.md

```diff
@@ -23,6 +41,71 @@ flutter:
         - asset: assets/fonts/abrilfatface/AbrilFatface-Regular.ttf
 ```
 
+### Second use the font
+
+Second use the font by delcaring the [`TextStyle`](https://docs.flutter.io/flutter/painting/TextStyle-class.html)
+constant and adding the font family used in the `pubsec.yaml`.
```

Comment:
What do you mean "the font family used in the pubspec" ? When I look at the example above, I don't see a mention of "font family".